views/partials/dependencies.hbs

```diff
@@ -47,6 +47,10 @@
         break;
     }
 	</script>
+  <!--[if IE]>
```

Comment:
preferably at the top with the other ie shims, and indented to match the style (tbh, the others are indented wrong too, so I'm more preferring that this be at the top with them, than it being indented right.